reconcile/saas_file_owners.py

```diff
@@ -99,6 +101,28 @@ def collect_baseline():
     return {'owners': owners, 'state': state}
 
 
+def collect_compare_diffs(current_state, desired_state):
+    compare_diffs = set()
+    for d in desired_state:
+        for c in current_state:
+            if d['saas_file_name'] != c['saas_file_name']:
+                continue
+            if d['resource_template_name'] != c['resource_template_name']:
+                continue
+            if d['environment'] != c['environment']:
+                continue
+            if d['cluster'] != c['cluster']:
+                continue
+            if d['namespace'] != c['namespace']:
+                continue
+            if d['ref'] == c['ref']:
+                continue
+            compare_diffs.add(
+                f"{d['url']}/compare/{c['ref']}...{d['ref']}")
```

Comment:
Cool that this works in gitlab and github :)

Reply:
i was surprised!